ARE/services/WebService/src/main/java/eu/asterics/mw/webservice/RestServer.java

```diff
@@ -75,8 +80,9 @@
     @Path("/restfunctions")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
+    @Description("Returns a list with all the available rest functions")
     public String getRestFunctions() {
-        String JSONresponse = ObjectTransformation.objectToJSON(ServerRepository.restFunctions);
+        String JSONresponse = ObjectTransformation.objectToJSON(ServerRepository.getInstance().createListOfRestFunctions());
```

Comment:
I would recommend using shorter names in future. Especially in cases like this, where we only have few lines of code, and no other variable are used nor namespace collisions easily possible. This would make the code easier to read.

Reply:
However the name should clearify what is done. Maybe an alternative would be `getRestFnList()`.